gkeys/gkeys/keyhandler.py

```diff
@@ -108,3 +111,27 @@ def key_search(self, args, first_match=False):
 
         self.logger.debug(_unicode("KeyHandler: key_search; keys = %s") % str(keys))
         return keys
+
+    def _is_expiring(self, results, days_limit):
+        is_expiring = False
+        for key in results:
+            for specs in results[key]:
+                if specs.days > days_limit*(-1) and specs.days < days_limit:
+                    is_expiring = True
+                    break
+        return is_expiring
```

Comment:
this function is not using anything from self, so it should get an @staticmethod decorator, remove self from the parameters...

but...
keyhandler class is initialized with the config instance, so it can look up days_limit for itself.  Then make the days_limit parameter optional, defaulting to the config value.  Which is imo the better option.